shared/src/test/scala/purecsv/safe/converter/ConverterSuite.scala

```diff
@@ -15,10 +15,12 @@
 package purecsv.safe.converter
 
 import org.scalatest.{FunSuite, Matchers}
+import purecsv.util.serializeAndDeserialize
 import shapeless.{::, Generic, HNil}
 
 import scala.util.Success
 
+case class Event(ts: Long, msg: String, user: Option[Int])
```

Comment:
These test case classes were moved outside of the class due to serialization liking to grab more than it ought to.